### DISCUSSION TITLE
Moving CI from AppVeyor to GitHub actions

### DISCUSSION THREAD

**Discussion by winterz** [libical/libical#648] [General]
AppVeyor has really slow turn-around time.  I wonder if github actions are faster?

would anyone like to take on the task of adding libical to github actions CI?
then I can remove the appveyor web hook

**Comment by tintou** [comment]
Created a first attempt here https://github.com/libical/libical/pull/649

**Comment by winterz** [comment]
merged.
Thanks so much.

I deleted the appveyor web hook
